platform-api/che-core-api-user/src/test/java/org/eclipse/che/api/user/server/UserServiceTest.java

```diff
@@ -269,6 +271,65 @@ public void shouldBeAbleToRemoveUser() throws Exception {
         verify(userDao).remove(testUser.getId());
     }
 
+
+    /**
+     * Check admin user is 'true' for isUserInRole' with admin role
+     * @throws Exception
+     */
```

Comment:
Do we really need javadoc here, i think that method declaration describes it enough